zerver/views/documentation.py

```diff
@@ -75,6 +75,9 @@ def get_path(self, article: str) -> Tuple[str, int]:
             http_status = 404
 
         path = self.path_template % (article,)
+        article_path = os.path.join(settings.DEPLOY_ROOT, "templates") + path
```

Comment:
Why is path added rather than included in the `os.path.join`?

Reply:
This is later used in the code as well. Quoting from there:
```
# The following is a somewhat hacky approach to extract titles from articles.
        # Hack: `context["article"] has a leading `/`, so we use + to add directories.
        article_path = os.path.join(settings.DEPLOY_ROOT, "templates") + context["article"]
```

Reply:
I see, OK; worth doing a comment then, because generally one should use the preferred combination functions.

(Maybe we want `urljoin` instead?)

Reply:
`urljoin` can only join two at a time, so anyway we would have to use multiple instances of it or combine it with `path.join`. Idts it's as clean a solution as we would expect.